src/components/MovieCard.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import { MovieDetails } from "./MovieDetails";
+import "./MovieCard.css";
+
+export const MovieCard = () => {
+  const [movieData, setMovieData] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const { id } = useParams();
+  // console.log("movie_id:", id);
+
+  useEffect(() => {
+    const api_key = "7a6dc3dad2658fff3044692170b4b00a";
```

Comment:
If you make an env file to put your key in you don't have the key exposed for the public.